charts/yorkie-argocd/README.md

```diff
@@ -0,0 +1,93 @@
+# yorkie-argocd
+
+Installs the yorkie-argocd, which provides GitOps based continuous delivery for yorkie cluster.
+
+## Prerequisites
+
+- Kubernetes 1.24+
+- Helm 3+
+
+## Get Helm Repository Info
+
+```bash
+helm repo add yorkie https://yorkie-team.github.io/yorkie/helm-charts
+helm repo update
+```
+
+_See [`helm repo`](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Helm Chart
+
+```bash
+# Install yorkie argocd helm chart
+helm install [RELEASE_NAME] yorkie/yorkie-argocd -n argocd --create-namespace
+
+# Update admin password
+## bcrypt(yorkie)=$2a$12$fJoZj9CnNyD5Yfi02nZh7.XcH4Ds9M.oftQOQDP5oytyra9cP6Dny
+kubectl -n argocd patch secret argocd-secret \
+  -p '{"stringData": {
+    "admin.password": "$2a$12$fJoZj9CnNyD5Yfi02nZh7.XcH4Ds9M.oftQOQDP5oytyra9cP6Dny",
+    "admin.passwordMtime": "'$(date +%FT%T%Z)'"
```

Comment:
What is the actual value of `admin.password`?

Reply:
it's on the commend above: `bcrypt(yorkie)=$2a$12$fJoZj9CnNyD5Yfi02nZh7.XcH4Ds9M.oftQOQDP5oytyra9cP6Dny`